bookstore/bookstore.py

```diff
@@ -1,29 +1,94 @@
+#return dictionary with {'name': name}
 def create_bookstore(name):
-    pass
-
+    rmotr_bookstore = {}
```

Comment:
There's a tidier way to initialize your dictionary here.
```
return {
        'name': name,
        'nationality': nationality,
        'authors': {},
        'books': {}
        }
```
Accomplishes the same thing as what you have here.